examples/tests/whiteboard.py

```diff
@@ -38,7 +38,7 @@ def test(self):
         result = ''
         for _ in range(0, iterations):
             result += data
-        self.whiteboard = base64.encodestring(result)
+        self.whiteboard = base64.encodestring(result.encode()).decode('ascii')
```

Comment:
yep

Reply:
Well if we require users to encode `whiteboard` in a specific format we're doing something wrong. The `whiteboard` should be lenient enough to avoid breakages in case some user provides `bytes`, strings or unicodes, perhaps even broken unicodes (we have the `avocado.utils.astring.string_safe_encode`, perhaps we can use that...)

Reply:
Well, the whiteboard is **currently** described as:

"Arbitrary string which will be stored in `$logdir/whiteboard` location when the test finishes."

The point here is what a "string" means.  In Python 2, it can mean bytes.  In Python 3, it can't.  Since we need compatibility on both Python 2 and 3, the imaginary writer of this test would need to pass a string indeed.

I see your point though, and we can discuss this further.